requirements.in

```diff
@@ -0,0 +1,10 @@
+funding-service-design-utils>=2.0.2,<2.1.0
+cssmin==0.2.0
+jsmin==3.0.1
+flask==2.3.2
+flask-assets==2.0
+govuk-frontend-jinja==2.7.0
+govuk-frontend-wtf==2.5.0
+gunicorn==20.1.0
+bs4
```

Comment:
Is this being used? If so maybe it can be in the test requirements only?

Reply:
govuk-frontend-jinja and bs4 are, if we're removing all code related to analytics cookies we can delete govuk-frontend-wtf. I don't think we're currently using gunicorn, although we might need it soon. I will remove for now.

Reply:
gunicorn will be used if its ever run in a docker container so I'd leave it in